andela/vale-ah-backend
# #164139774 Create new recipe

**What does this PR do?**
Create new recipe.

**Description of Task to be completed?**
- write unit test
- defined the recipe schema
- implement create article

**How should this be manually tested?**
- clone this repository
- run `npm install`
- start the dev server, 'npm run devstart`
- open up postman and generate an existing user's token using the endpoint, 'localhost:{PORT}/user/login. Ensure the email and password of the user is provided.
- add the token to the req.headers with 'authorization' as the key and the endpoint to 'localhost:{PORT}/recipe

**Any background context you want to provide?**
- New recipes can only be created by a registered user

**What are the relevant pivotal tracker stories?**
#164139774

**Screenshots (if appropriate)**

[#164139774]
